x-pack/plugins/security_solution/public/common/components/guided_onboarding_tour/README.md

```diff
@@ -0,0 +1,140 @@
+## Security Guided Onboarding Tour
```

Comment:
stole this from my PR description, thought it would be good to commit

Reply:
Nice, thanks! Can you point this out to others who are developing the other steps so they can update it for their steps as well? When the work is done, I guess you'll remove the "weirdness and creativity for reasons stuff 😂"?